python/mxnet/contrib/onnx/_import/op_translations/_add.py

```diff
@@ -0,0 +1,25 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+# coding: utf-8
+
+def _add(op_name, attrs, inputs):
+    new_attr = {}
+    if 'broadcast' in attrs and attrs['broadcast']==1:
+        return 'broadcast_add', new_attr, inputs
```

Comment:
why not create the operator here directly?

Reply:
If I create the operator here, then the same create operator command has to be repeated for every operator translation in every method. 

Probably more cleaner and elegant to do it once in the caller method.